style.css

```diff
@@ -313,3 +318,214 @@ textarea {
   color: #42526e;
   margin-bottom: 147px;
 }
+
+/* -----------------------------------media */
+
+@media screen and (min-width: 1080px) {
+  body {
+    background-image: url(imege/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 100%;
+  }
+
+  nav {
+    height: 32px;
+  }
+
+  .nav-list {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    gap: 40px;
+    list-style: none;
+    font-weight: 500;
+    height: 100px;
+    padding-bottom: 6rem;
+  }
+
+  nav ul li a {
+    color: #42526e;
+  }
+
+  .nav-list :hover {
+    color: #36b37e;
+    transition: 0.3s;
+  }
+
+  .nav-list :active {
+    color: #42526e;
+  }
+
+  .logo,
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: none;
+    padding: 160px 0 0 0;
+    margin: 0 270px;
+    justify-content: center;
+  }
+
+  #head-1 {
+    height: 114px;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  p {
+    height: 96px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #42526e;
+  }
+
+  .icon {
+    list-style: none;
+    padding-top: 40px;
+    padding-bottom: 40%;
+  }
+
+  .action {
+    visibility: hidden;
+  }
+
+  .first-page h2 br,
+  #indicator {
+    display: none;
+  }
+
+  footer h2 {
+    width: 436px;
+    height: 256px;
+    font-weight: 700;
+    font-size: 28px;
+    line-height: 52px;
+    margin: 50px 24px 50px 135px;
+    text-align: justify;
+    text-justify: inter-character;
+  }
+
+  .first-page h2 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    padding-bottom: 52px;
+  }
+
+  .Proj-container,
+  .skills {
+    padding: 0 135px;
+    justify-content: space-between;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .proj-card {
+    width: 369px;
```

Comment:
- [ ] Well done on using display:grid to show card but please move card in the middle on the grid because in desktop screen right side is having more space than left size. For example you can use margin: auto and it will be fixed. Keep coding 💪